ebuild-writing/functions/src_install/text.xml

```diff
@@ -92,48 +92,6 @@ src_install() {
 </body>
 </section>
 
-<section>
-<title>Easy Installs</title>
-<body>
-<p>
-Often, especially with autotools-powered packages, there is a <c>Makefile</c>
-<c>install</c> target which will honour the <c>DESTDIR</c> variable to tell it to
-install to a non-root location. If possible, this should be used:
-</p>
-
-<codesample lang="ebuild">
-	emake DESTDIR="${D}" install
-</codesample>
-
-<note>
-<c>emake</c> should be used to parallelise here. Some installs are
-not designed to be parallelised, use <c>emake -j1</c> or <c>make</c>
-if you hit an error.
-</note>
-
```

Comment:
This is way too intrusive. IMHO the section with the "emake install" example should remain in place.